cmd/restic/cmd_backup.go

```diff
@@ -115,11 +117,17 @@ func init() {
 	f.StringVar(&backupOptions.ExcludeLargerThan, "exclude-larger-than", "", "max `size` of the files to be backed up (allowed suffixes: k/K, m/M, g/G, t/T)")
 	f.BoolVar(&backupOptions.Stdin, "stdin", false, "read backup from stdin")
 	f.StringVar(&backupOptions.StdinFilename, "stdin-filename", "stdin", "`filename` to use when reading from stdin")
+	err := f.MarkDeprecated("stdin-filename", "use --set-path")
+	if err != nil {
+		// MarkDeprecated only returns an error when the flag could not be found
+		panic(err)
+	}
+
 	f.Var(&backupOptions.Tags, "tag", "add `tags` for the new snapshot in the format `tag[,tag,...]` (can be specified multiple times)")
 
 	f.StringVarP(&backupOptions.Host, "host", "H", "", "set the `hostname` for the snapshot manually. To prevent an expensive rescan use the \"parent\" flag")
 	f.StringVar(&backupOptions.Host, "hostname", "", "set the `hostname` for the snapshot manually")
-	err := f.MarkDeprecated("hostname", "use --host")
```

Comment:
Hi @aawsome 

Just noticed this line, where assignment was changed to equality.  Was this intended?

Reply:
Isn't that just a change from declaration assignment to plain assignment? Since `err` is now declared further up in line 120.

Reply:
exactly!

Reply:
> Isn't that just a change from declaration assignment to plain assignment? Since err is now declared further up in line 120.
Thanks for the explanation.  Indeed my mistake - I assumed Go used Pascal syntax.